src/Optimizely/Event/Builder/EventBuilder.php

```diff
@@ -102,7 +103,8 @@ private function getCommonParams($config, $userId, $attributes)
             foreach ($attributes as $attributeKey => $attributeValue) {
                 $feature = [];
                 // Do not discard attribute if value is zero or false
-                if (!is_null($attributeValue)) {
+                // Omit attributes that are not supported by the log endpoint.
```

Comment:
The comment _ // Do not discard attribute if value is zero or false is no longer valid at this place. Remove